src/main/kotlin/no/nav/familie/ef/sak/behandling/oppgaverforferdigstilling/OppgaverForFerdigstillingController.kt

```diff
@@ -0,0 +1,32 @@
+package no.nav.familie.ef.sak.behandling.oppgaverforferdigstilling
+
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.UUID
+
+@RestController
+@RequestMapping("/api/oppgaverforferdigstilling")
```

Comment:
anbefaler å bruke `-` for å bryte opp url i ord;
`.../oppgaver-for-ferdigstilling`